### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getPath' of undefined

### ISSUE THREAD

**Issue by autopawn** [andrewrynhard/atom-latex-plus#19]
[Enter steps to reproduce below:]
1. Create an empty .tex file.
2. Execute build command.

**Atom Version**: 0.187.0
**System**: linux 4.1.5-100.fc21.x86_64
**Thrown From**: [latex-plus](https://github.com/andrewrynhard/atom-latex-plus) package, v0.7.0
### Stack Trace

Uncaught TypeError: Cannot read property 'getPath' of undefined

```
At /home/apawn/.atom/packages/latex-plus/lib/project.js:125

TypeError: Cannot read property 'getPath' of undefined
  at Project.edit (/home/apawn/.atom/packages/latex-plus/lib/project.js:125:38)
  at atom-workspace.commands.atom.commands.add.latex-plus:edit (/home/apawn/.atom/packages/latex-plus/lib/main.js:97:23)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:246:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:182:20)

```
### Commands

```
     -1:08.6 application:new-file (ul.list-inline.tab-bar.inset-panel)
  2x -0:53.4 window:increase-font-size (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:51.3 tree-view:collapse-directory (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:47.9 editor:toggle-line-comments (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:47.3 latex-plus:edit (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  3x -0:45.7 tree-view:collapse-directory (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:37.2 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:34.4 core:confirm (atom-text-editor.editor.mini)
     -0:34.3 application:zoom (atom-text-editor.editor.is-focused)
     -0:32.8 command-palette:toggle (atom-text-editor.editor.is-focused)
  2x -0:31.2 core:backspace (atom-text-editor.editor.mini)
     -0:26.6 core:confirm (atom-text-editor.editor.mini)
     -0:26.5 window:reset-font-size (atom-text-editor.editor.is-focused)
     -0:04.4 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:04.4 core:cancel (atom-text-editor.editor.is-focused)
     -0:00.7 latex-plus:edit (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "minimap"
    ],
    "projectHome": "/home/apawn/Documentos"
  },
  "latex-plus": {
    "bibtexEnabled": true,
    "texBin": "/usr/bin/latex"
  }
}
```
### Installed Packages

``` coffee
# User
language-latex, v0.6.1
latex-plus, v0.7.0

# Dev
No dev packages
```

**Comment by andrewrynhard** [comment]
@Autopawn: From the looks of the stack trace you invoked the `edit` command. Could you try running the `compile` command from the `Packages` menu within atom?

**Comment by andrewrynhard** [comment]
@Autopawn: I cannot seem to reproduce this error. Are you able to use this package at all, or does this error keep happening?

**Comment by autopawn** [comment]
I wasn't able to run the package at all. The problem with the edit command keeps happening.

The Compile command gets stuck (I just see the "Compiling..." text on the bottom bar). But I noticed that the first time that I run it (first time on a window), an error message said that a Latex installation had to be specified on the latex plus settings, then a "save as" window randomly poped-up, and I selected a file, but the file was left blank and the "Compiling" text was there again.
The errors keep appearing even when I set the latex directory on the package configuration.

May it be because I didn't manually wrote the "Project Configuration" file that the README talks about?
Anyway, It may be because I am using Atom on Linux, everything is buggy for me. [SFMBE](http://es.urbandictionary.com/define.php?term=SFMBE)
